src/legacy/core_plugins/region_map/index.ts

```diff
@@ -28,7 +28,6 @@ const regionMapPluginInitializer: LegacyPluginInitializer = ({ Plugin }: LegacyP
     require: ['kibana', 'elasticsearch', 'visualizations', 'interpreter', 'data'],
     publicDir: resolve(__dirname, 'public'),
     uiExports: {
-      styleSheetPaths: resolve(__dirname, 'public/index.scss'),
```

Comment:
I'm not sure if I just can remove this (since we don't have any particular styles inside region map), but it works without warnings or exceptions.
@timroes @spalger could you please confirm ?

Reply:
If we have no style files anymore, that should be simply removable.